.gitignore

```diff
@@ -80,6 +80,7 @@ scikits
 !np_datetime_strings.c
 *.c
 *.cpp
+*.pxi
```

Comment:
AFAIK `.pxi` are just 'include' files, so they are not auto-generated by definition.